de.sebastianruziczka.looksyk.yml

```
@@ -0,0 +1,58 @@
+app-id: de.sebastianruziczka.looksyk
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
```

```suggestion
runtime-version: '24.08'
```

One doesn't simply mismatch runtime version.

---

Done (with comment on other thread)